### PR TITLE
modify license choice language

### PR DESCRIPTION
**Pull request by andrewsu**
Given that the choice between CC0 and CC-BY is a nuanced one with many pros and cons on both sides of the issue, I offer three suggestions for this document:

1. linking to https://github.com/OBOFoundry/OBOFoundry.github.io/issues/285 where many issues are explicitly discussed
2. removing the explicit recommendation of CC-BY
3. adding a request for attribution in all cases regardless of license (following [this pattern](http://www.dancohen.org/2013/11/26/cc0-by/))

I of course understand that this policy is ultimately under the purview of the Editorial WG, but I've formulated this as a pull request just to propose something specific.

### CODE REVIEW

**Comment by cmungall** [comment]
+1

Other members of @OBOFoundry/obo-admin should vote

**Comment by bpeters42** [comment]
This proposal of 'ethical attribution' writes out much better than I could what my gut feeling was we should be doing. I enthusiastically support this.

**Comment by lschriml** [comment]
Enthusiastic 
+1 
Lschriml

**Comment by mellybelly** [comment]
I think this is a good idea. Linking to the ticket for discussion of the issues, but I think the primary link should be to our citation policy http://www.obofoundry.org/docs/Citation.html, we may want to update.

**Comment by zhengj2007** [comment]
+1

If we made the changes, we'd also announce it on obo-discuss list.

**Comment by kltm** [comment]
I would agree with the link to the tracker conversation and request of attribution regardless of license choice.

However, while I agree that there should be no explicit recommendation of CC-BY, I think it would be a very good idea to help people who are new licensing questions with a bit more direction in open licensing, and suggest CC and CC licenses as a starting point (as they do have very good discussion and are quite robust).

Possibly (overly strong) verbiage along the lines of:

We recommend that new ontologies carefully consider the choice of license. For example, [Creative Commons](https://creativecommons.org/) licenses like [CC0]() and [CC-BY]() are popular open and understandable licenses. Many pros and cons of each license are laid out in [this discussion](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/285).

I think directed discussion may be important to keep people on board and concerned about the issue--while the thread is interesting, I think it may be a bit much for people who are new to thinking about licenses and leading to a sensible default may be valuable.

**Comment by andrewsu** [comment]
FWIW, @kltm's suggested revision gets a strong thumbsup from me. I [added]( https://github.com/andrewsu/OBOFoundry.github.io/commit/cbabca526c5750dec2581075b374b0f9f3983895) the change to the pull request...

**Comment by andrewsu** [comment]
and realized I overlooked @mellybelly's suggestion, which I think is also great...  [added](https://github.com/andrewsu/OBOFoundry.github.io/commit/408ee8729e4a2b8d238e4af57459ee7492eb3c0a)...

**Comment by jamesaoverton** [comment]
+1 for the latest version

**Comment by mcourtot** [comment]
+ 1 with suggestion of adding something along the lines of "The licence should be clearly stated in the ontology file."

**Comment by andrewsu** [comment]
I agree with @mcourtot and am happy to add something to the PR. But I'm wondering what the right protocol here is. With every additional change, do people need to re-vote?  (Is there a formal voting process by the Editorial WG?) It seems there is broad support for the changes, so would like to know the best way to wrap this up...

**Comment by cmungall** [comment]
For additions that are informative and clearly improve/clarify I don't think we need to re-vote. For normative changes such as @mcourtot's in general it would be good to at least give people a chance to veto.

However until we can get a sensible policy in our CONTRIBUTING.md (hopefully in a month or so) let's proceed pragmatically. Given the  https://tools.ietf.org/html/rfc2119 definition of SHOULD the proposed addition is non-binding and I don't foresee any objections, so let's do it.

**Comment by ramonawalls** [comment]
+1

------------------------------------------------------
Ramona L. Walls, Ph.D.
Senior Scientific Analyst, CyVerse, University of Arizona
Research Associate, Bio5 Institute, University of Arizona

**Comment by andrewsu** [comment]
done!  added a line that reflects @mcourtot's suggestion...

**Comment by lschriml** [comment]
+1 for the latest version

File: principles/fp-001-open.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ OBO Foundry ontologies are resources for the entire biological and biomedical co
 
     OR released into the public domain under [CC0](https://creativecommons.org/publicdomain/zero/1.0/) (which is not a license). Note that CC0 specifies that the creators of an ontology waive, to the extent that they legally can be, all rights and place the ontology in the public domain. It does not prevent them from requesting that the ontology be properly credited and cited, but prevents any legal recourse if it is not credited.
```
Comment (hlapp) [inline principles/fp-001-open.md]: _to the extent that they legally can be_: typo, should remove _be_

**Pull request merged**